kqueen_ui/blueprints/ui/templates/ui/partial/selectform.html

```diff
@@ -43,7 +43,19 @@
   $.validator.addMethod("yamlfile", validateYamlFileField, $.validator.format("Test validation"));
   $.validator.addMethod(
     "cidr", validateFieldIsIpCidr, $.validator.format("Value doesn't match CIDR format."));
+  $.validator.addMethod('uuid', validateFieldIsUuid, 'Entered value is not a valid UUID.');
 
+  // Prevent two clicks after hiding (https://github.com/twbs/bootstrap/issues/16732)
+  $('body').on('hidden.bs.popover', function (e) {
+    $(e.target).data("bs.popover").inState.click = false;
+  });
+  $("i.mdi-help-circle").popover({html: true, placement: "auto right"})
+    .click(function () {
+      // Hide help after 5 seconds
```

Comment:
Why do you need to hide it? Why after 5 sec? If you feel it should be hidden, then it's better to hide it when it loses focus (when a user clicks somewhere outside of it)